### ISSUE TITLE
Make the 403 & 404 pages more user-friendly

### ISSUE THREAD

**Issue by jmcameron** [third-culture-software/bhima#5125]
We have two HTML pages for 403 and 404 errors:
```
   client/src/modules/errors/403.html  
   client/src/modules/errors/404.html
```
We should replace the '403' and '404' numbers on those pages to more helpful messages (with translation).

**Comment by jniles** [comment]
I agree!  We might also want to indicate what page they were going to, such as "you don't have access to Cash Payments".   And maybe provide a link back to the previous page - "return to Invoice Registry?"

**Comment by jniles** [comment]
See also https://github.com/third-culture-software/bhima/issues/520.
